### ISSUE TITLE
Benefits over Twig and Blade?

### ISSUE THREAD

**Issue by leek** [pagekit/razr#4]
It would be helpful if the README explained why anyone would want to use this over well established projects like Twig and Blade.

**Comment by fer-ri** [comment]
+1

**Comment by fer-ri** [comment]
So far i just see about @set function thats missing from blade

**Comment by jaywilliams** [comment]
Yes, I too am wondering the very same thing. The Twig codebase is very stable and well maintained.

**Comment by acydburn** [comment]
+1 - Would be nice to hear the reasoning behind it... at least you guys must've had heated discussions regarding this. ;)

**Comment by nbpalomino** [comment]
- 1 - This is a interesting topic... Im very in love with Twig, but I always think why make another templating system as Razr and Blade...

**Comment by eightyfive** [comment]
+1 - Twig is so well-established & powerful. Maybe make the template engine configurable?

**Update**
It seems like it's already well-done & configurable:
https://github.com/1up-lab/pagekit-twig/blob/master/src/Provider/TwigProvider.php#L41

**Comment by nbpalomino** [comment]
After months checking Razr... I could say Twig > Razr > Blade.

Razr is such a good template engine, its easy, clean and configurable; YooTheme guys you make a great job with Razr just need a better doc's and a standalone website like Twig...

**Comment by surfmuggle** [comment]
@nbpalomino  By "twig > razor" do you mean that twig is better than razor?

What is php razor missing?

**Comment by surfmuggle** [comment]
@leek One reason for us would be if php razor and asp.net razor were identical that we could modernize the frontend (all echo now) by using razor and have synergies for colleagues already familiar with it and later could [switch from php to c#](https://peachpie.io)
